tpgtools/types/constants.go

```diff
@@ -0,0 +1,12 @@
+package types
```

Comment:
For both `types` and `utils`, exactly one file within those packages should have a package comment. I've used otherwise empty files matching the package name, personally.

https://golang.org/doc/effective_go#commentary

Reply:
will add